packages/generator/src/v2.0/generatePerQueryDataFiles.ts

```diff
@@ -116,8 +120,9 @@ async function generateV2QueryFile(
   await fs.writeFile(
     path.join(outDir, `${query.shortApiName}.ts`),
     await formatTs(`
-        import type { QueryDefinition , VersionBound} from "@osdk/api";
-        import type { QueryParam, QueryResult } from "@osdk/api";
+        import type { QueryDefinition , QueryParam, QueryResult, VersionBound} from "${
```

Comment:
I'm not sure why these were split up before, just combined. If this was done for a reason lmk and I'll revert

Reply:
Oversight probably